dts/arm/adi/max32/max32655.dtsi

```diff
@@ -75,5 +103,25 @@
 			clock-source = <ADI_MAX32_PRPH_CLK_SRC_IBRO>;
 			status = "disabled";
 		};
+
+		timer4: timer@40080c00 {
```

Comment:
Timer4/5 generally low power timers, the name of the node shall be lptimer0/1 as device UG.

Reply:
Done